WIP: Remove deprecated features (#189)
neuralet/smart-social-distancing: README.md

 ***NOTE*** Most of the endpoints update the config file given in the Dockerfile. If you don't have this file mounted (see section [Persisting changes](#persisting-changes)), these changes will be inside your container and will be lost after stopping it.

### Interacting with the processors' generated information

#### Generated information
The generated information can be split into 3 categories:

>   - `Raw data`: This is the most basic level of information. It only includes the results of the detector, classifier, tracker, and any configured post-processor step. All the video information generated (such as the live feed) is **stored inside the edge device and is never sent outside it**. 

Reviewer: ```suggestion
  - `Raw data`: This is the most basic level of information. It only includes the results of the detector, classifier, tracker, and any configured post-processor step. All the video information generated (such as the live feed) is **stored inside the edge device and is never sent to cloud or any other storage**. 
```
It might be sent to the browser of the users

>   - `Metrics data`: **Generated in Lanthorn's cloud dashboard, only generated if you register a processor in your account.**. These include metrics related to occupancy, social-distancing, and facemask usage; aggregated by hour and day. When you enabled that functionality, some raw data needs to be sent from your device to our cloud engine. However, **only numeric and statistical data is sent. No video or personal information is sent from your device to Lanthorn's servers.**

Reviewer: ```suggestion
  - `Metrics data`: **Generated in Lanthorn's cloud dashboard, only generated if you register a processor in your account.**. These include metrics related to occupancy, social-distancing, and facemask usage; aggregated by hour and day. When you enabled that functionality, some raw data needs to be sent from your device to our cloud engine. However, **only numeric and statistical data is sent. No video, images or personal identifiable information is sent from your device to Lanthorn's servers.**
```

>   - `Notifications`: Situations that require an immediate response (such as surpassing the maximum occupancy threshold for an area) and need to be notified ASAP. The currently supported notification channels are email and slack. Same that the metrics, this feature is supported by Lanthorn's cloud engine.

Reviewer: ```suggestion
  - `Notifications`: Situations that require an immediate response (such as surpassing the maximum occupancy threshold for an area) and need to be notified ASAP. The currently supported notification channels are email and slack. Same as the metrics, this feature is supported by Lanthorn's cloud engine.
```
